myproject/page1.php

```diff
@@ -10,8 +16,24 @@
 
 }
 
-var_dump($list);
+//var_dump($list);
 
 fclose($handle);
-
-?>
\ No newline at end of file
+?>
+<?php if (count($list) > 0): ?>
+<table style="color: #5892D5">
+  <thead>
+    <tr>
+      <?php foreach ($list as $row): array_map('htmlentities', $row); ?>
+    </tr>
+  </thead>
+  <tbody>
+	<tr>
+      <td><?php echo implode('</td><td>', $row); ?></td>
+    </tr>
+<?php endforeach; ?>
```

Comment:
It would be better to add indent before closing foreach as the same as before opening foreach. They should be with the same indent.